picard/ui/theme.py

```diff
@@ -75,6 +108,8 @@ def setup(self, app):
 
     @property
     def is_dark_theme(self):
+        if self._loaded_config_theme != UI_THEME_DEFAULT:
+            return self._loaded_config_theme == UI_THEME_DARK
```

Comment:
This should be

```
@property
    def is_dark_theme(self):
        if self._loaded_config_theme == UI_THEME_DARK:
            return True
        elif self._loaded_config_theme == UI_THEME_LIGHT:
            return  False
        else:
            return self._dark_theme
```

Otherwise it breaks the heuristic to detect whether the theme in use has dark background or not.